src/languages/en-US/commands/misc.json

```
@@ -358,6 +382,20 @@
 			"A E S T H E T I C"
 		]
 	},
+	"viewBirthdayDescription": "View a user's birthday.",
+	"viewBirthdayExtended": {
+		"extendedHelp": "Birthdays are stored per-server. If you don't see yours, you probably haven't used the `set-birthday` command in this server yet!",
+		"usages": [
+			"@User"
+		],
+		"examples": [
+			"@Stitch#4094"
+		]
```

```suggestion
		"usages": [
			"User"
		],
		"extendedHelp": "Birthdays are stored per-server. If you don't see yours, you probably haven't used the `{{prefix}}set-birthday` command in this server yet!",
		"explainedUsage": [
			[
				"User",
				"The user for whom to view the birthday"
			]
		],
		"examples": [
			"@Stitch#4094"
		]
```

Same deal as before with `{{prefix}}`. Also a full block review because overall `usages` is above `extendedHelp`

---

FYI I don't need to add a prefix here bc the help command already does that.